spec/scoreSpec.js

```diff
@@ -0,0 +1,36 @@
+describe("Score", function(){
+
+  var score;
+  beforeEach(function() {
+    score = new Score();
+  });
+
+  describe('roll counting', function(){
+
+    it('it can roll', function(){
+      expect(score.rolls.length).not.toEqual(0);
+    });
+
+    it('it can hit pins', function(){
+      score.rolls[0].ball(4)
```

Comment:
Law of Demeter - talk to your friends, but don't talk to your friend's friends. In other words, don't chain dots!

Score -> Roll -> Ball method